src/main/java/net/minecraftforge/client/MinecraftForgeClient.java

```diff
@@ -118,4 +118,10 @@ public static ChunkCache getRegionRenderCache(World world, BlockPos pos)
         int z = pos.getZ() & ~0xF;
         return regionCache.getUnchecked(Pair.of(world, new BlockPos(x, y, z)));
     }
+
+    public static void clearRenderCache()
+    {
+        regionCache.invalidateAll();
```

Comment:
Is there a property of `World` that we can safely retain as a cache key instead, so we don't have to worry about invalidating this cache?